compose.yaml

```diff
@@ -0,0 +1,15 @@
+services:
+  resourcematch-api:
+    build:
+      context: ./resourcematch-api
+    command: ["uvicorn", "main:app", "--host", "0.0.0.0", "--port", "80", "--reload"]
+    environment:
+      - OPENAI_API_KEY
```

Comment:
@MReyna12 , @P-Gill97 - I went ahead and set things up to where (in theory) in the future if you set this environment variable in your terminal and run `docker compose -d`, it'll pass it through to the container. The `openai` library will automatically pick that up when set, so things should just work from there for local dev/testing (see example in [k12-557](https://github.com/openstax/k12/issues/557) issue description).